redirects/17.0.txt

```diff
@@ -64,6 +64,7 @@ applications/marketing/marketing_automation/getting_started/target_audience.rst
 applications/marketing/marketing_automation/getting_started/testing_running.rst applications/marketing/marketing_automation/testing_running.rst           # getting_started/* --> marketing_automation/*
 applications/marketing/marketing_automation/getting_started/workflow_activities.rst applications/marketing/marketing_automation/workflow_activities.rst   # getting_started/* --> marketing_automation/*
 applications/marketing/surveys/time_random.rst applications/marketing/surveys/questions.rst   # time_random --> questions
+applications/marketing/sms_marketing/essentials.rst applications/marketing/sms_marketing.rst 								  # /marketing/sms_marketing/essentials --> /marketing/sms_marketing
```

Comment:
Need to add redirects for all moved and deleted files :)

Also please only use spaces, never tabs, for spacing and indenting in `.rst` and redirect `.txt` files :)
```suggestion
applications/marketing/sms_marketing/essentials.rst applications/marketing/sms_marketing.rst                                                              # /marketing/sms_marketing/essentials --> /marketing/sms_marketing
applications/marketing/sms_marketing/essentials/sms_essentials.rst applications/marketing/sms_marketing.rst                                               # /marketing/sms_marketing/essentials --> /marketing/sms_marketing
applications/marketing/sms_marketing/essentials/sms_campaign_settings.rst applications/marketing/sms_marketing.rst                                        # /marketing/sms_marketing/essentials --> /marketing/sms_marketing
applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst applications/marketing/sms_marketing/mailing_lists_blacklists.rst            # /marketing/sms_marketing/essentials --> /marketing/sms_marketing
applications/marketing/sms_marketing/pricing.rst applications/marketing/sms_marketing/pricing_and_faq.rst                                                 # /marketing/sms_marketing/pricing --> /marketing/sms_marketing
applications/marketing/sms_marketing/pricing/pricing_and_faq.rst applications/marketing/sms_marketing/pricing_and_faq.rst                                 # /marketing/sms_marketing/pricing --> /marketing/sms_marketing
```